test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Converters/IconUrlToImageCacheConverterTests.cs

```diff
@@ -83,5 +98,160 @@ public void Convert_WithValidImageUrl_DownloadsImage()
             Assert.NotSame(DefaultPackageIcon, image);
             Assert.Equal(iconUrl, image.UriSource);
         }
+
+        [Fact(Skip = "Runs only on Windows Desktop with WPF support")]
+        public void Convert_EmbeddedIcon_HappyPath_LoadsImage()
+        {
+            using (var testDir = TestDirectory.Create())
+            {
+                // Create decoy nuget package
+                var zipPath = Path.Combine(testDir.Path, "file.nupkg");
+                CreateDummyPackage(zipPath);
+
+                // prepare test
+                var converter = new IconUrlToImageCacheConverter();
+
+                UriBuilder builder = new UriBuilder(new Uri(zipPath, UriKind.Absolute))
+                {
+                    Fragment = "icon.png"
+                };
+
+                output.WriteLine($"ZipPath {zipPath}");
```

Comment:
I like the output statements, but I'm not sure if this is intentional or standard for other tests? Curious because I may leave some useful output statements in my tests as well.

Reply:
It is intentional. It helps debug the tests in the CI. Others are using it:

https://github.com/NuGet/NuGet.Client/blob/6da9b6f04cf0198d89ff90cba64ad6e5c8027dcd/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs#L31